openkfw/TruBudget
# Create all initial letters uniformly small for workflow item

### Checklist




### Description

frontend:Standardize capital letter in translations

### How to test

1. Login
2. Open any subproject
3. Start creating new workflow item
4. Check translations in all languages




Closes #1887
